rosdoc2/verbs/default_config/__init__.py

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
Add copyright header

Reply:
I added a header, not sure if it is expected to dedicate copyright to OSRF, that can be done if desired.